packages/stems/src/components/Button/Button.module.css

```diff
@@ -46,6 +65,10 @@
   width: var(--unit-4);
 }
 
+.button.tiny .icon svg {
```

Comment:
Noticed with the new stories rendering all sizes that the icons on the tiny variant are actually way too big. We only use it in one place (track edit modal)